### DISCUSSION TITLE
QBO in ozone forcing in CMIP7

### DISCUSSION THREAD

**Discussion by JamesAnstey** [PCMDI/input4MIPs_CVs#22] [General]
The CMIP6 historical experiment ozone forcing included a quasi-biennial oscillation (QBO) component in the stratosphere. [This paper](https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2023GL104401) shows that the QBO (equatorial stratospheric zonal-mean zonal wind) tends to synchronize across realizations when a model uses this forcing, and doesn't synchronize for models using prognostic ozone. 

The question for discussion here is: should the CMIP7 ozone forcing include imposed interannual variability in ozone (as the CMIP6 forcing did), or should the QBO signal in ozone be removed?

**Comment by znichollscr** [comment]
@hegglinm any thoughts here?

**Comment by sosprey** [comment]
This is an important issue of relevance to other groups (e.g. WCRP [EPESC](https://www.wcrp-climate.org/epesc)) where attribution of external forcing and internal variability is of key importance. Clearly, having an internal mode of variability synchronise ensemble members in attribution experiments (and other situations) is something to avoid if possible. A transparent discussion on this issue, involving relevant groups involved in the creation of ozone datasets for CMIPx and other MIPs would be timely.

**Comment by JamesAnstey** [comment]
According to the [delivery summary table](https://input4mips-cvs.readthedocs.io/en/latest/database-views/input4MIPs_delivery-summary.html) that I found linked from the [docs](https://input4mips-cvs.readthedocs.io/en/latest/database-views/), historical ozone is planned for the end of May. Is any appreciable QBO signal expected in the dataset, and if so is the intention to leave it in or to remove it? @hegglinm @znichollscr @ccmi1-test

**Comment by ccmi1-test** [comment]
A common dataset of tropical winds, quite similar to the usual FUB dataset derived from radiosondes but extended in time beyond the available observations by cutting and pasting sections, has been provided to the modelling groups that are running a simulation to contribute to the CMIP7 ozone dataset. Whether a particular model has an internally-generated QBO or not, the goal is that all models have a synchronized QBO by nudging to the provided wind dataset and then the QBO signal in ozone will be removed by linear regression - or some other statistical treatment. If I am not mistaken, the idea is to provide versions with and without a QBO signal.

**Reply by JamesAnstey** [reply]
Great, thanks for the update @ccmi1-test. How many models are contributing? (Sorry if this is in the docs and I missed it.)

**Reply by ccmi1-test** [reply]
There are three models, as I understand it. WACCM, EMAC and CMAM. We are considering a 25% retaliatory tariff against the American model though.  ;-) More seriously, if you have any comments on the method let us know. Michaela is the one who will bring it all together and I am not sure if she is thinking about fitting the QBO individually for each model and then merging or fitting on the final dataset.

**Comment by hegglinm** [comment]
Hi @JamesAnstey, @sosprey, 

Thanks for your comments, and thanks to David (@ccmi1-test) for answering the questions (mostly correctly). 

Please note that I'm currently working under the  hypothesis that, for the historical past, we should aim for an ozone forcing that reflects **the true atmospheric state** as closely as possible - that is including the observed QBO. The reason is that we want to attribute observed changes using the best information available. For models that generate their own QBO (to my knowledge still only a few, with a properly resolved stratosphere) this will arguably pose a problem, as shown in the papers you cited above. However, for all the other models, incorporating the observed QBO would simply be an additional improvement. The same 'philosophy' applies when we add in the volcanic signals or biomass burning emissions that have clear ENSO signals in there over the observed past.  

For the PI control, I will offer a 20-year-long timeseries (with QBO) that should be repeated continuously for as long as the simulations are performed for. I will, in addition, provide a cilmatology of 12 months (with the QBO signal averaged out) for those models that do generate a QBO themselves. 

For the future, this presents an inherently unsolvable problem unless interactive chemistry is included. Prescribing a QBO synchronises variability, whereas omitting it alters the nature of the variability itself — potentially leading to overestimated trends in both variability and change.

Note, I am currently not intending to produce an ozone forcing without QBO before early next year and only if I hear that more people would want it. Please let me know if you were to use it (I understand that CanESM uses its own ozone anyway?)

Thanks much, 
Michaela

**Reply by durack1** [reply]
Thanks for the heads up @hegglinm!

This is a new approach to what we've had in the past. For all piControl datasets, we've typically observed a climatological seasonal cycle, rather than a quantification of variability modes. I know that the solar dataset, at least the recommended climatology for piControl use, is "fixed" so no variability. I do not believe that any other piControl recommended forcing dataset will provide an estimate of unforced variability; rather, these will be inherent in the historical timeseries only.

How "realistic" is the modelled QBO in contrast to our best guess obs? I am curious if the approach you recommend may infact be ***imposing*** a model bias, rather than fixing one?

**Comment by JamesAnstey** [comment]
Thanks @hegglinm & @durack1. So, the rationale for including a QBO signal in ozone in the ozone forcing dataset is that any  attribution of observed changes to the ozone forcing is better posed if that ozone forcing includes major signals of natural variability in ozone? In which case, for the historical ozone forcing:

- Nudging tropical winds toward the observed QBO (as described by @ccmi1-test) in all the models used to generate the dataset (WACCM, EMAC and CMAM) should produce a QBO ozone signal, synchronized across them.
- Running in AMIP mode using observed SSTs (is that what all the models do?) should produce an ENSO ozone signal, synchronized across them.

Is that accurate, @hegglinm and @ccmi1-test? And, what is done for piControl - in the 20-year timeseries ozone forcing, will it have a QBO component but no ENSO component?

If there will be a choice of which ozone dataset to use for piControl (climatological vs. 20-year repeated) then will models need to document in their EMD description which one was used?

**Reply by hegglinm** [reply]
@JamesAnstey, your understanding is correct! 

And yes - modellers should document exactly of how they implement the different options we would offer. The 20-year timeseries will only be a suggestion for models that have no internally generated QBO, just so to kick off some stratospheric variability. However, if not convinced the option will now be offered to everybody to simply use a climatological field.

**Reply by durack1** [reply]
> * Running in AMIP mode using observed SSTs (is that what all the models do?) should produce an ENSO ozone signal, synchronized across them.

@JamesAnstey I believe these data will be used across numerous experiments, not just the atmosphere-only AMIP, so coupled DECK piControl in particular have never had any variability modes imposed, at most it's been a seasonal/annual climatology. Not to say it isn't useful to include everything, we have a lot of forcing uncertainty to account for, it's just we need to make sure that we are clearly configuring and comparing apples to apples across sims (those which use this dataset, and those who don't)

**Reply by vnaik60** [reply]
Out of curiosity, do we know how many models will use this ozone dataset to drive their CMIP7 simulations? Do we also know how many of these include internally generated QBO versus not?

**Comment by hegglinm** [comment]
No! I tried to get such information from CMIP6, e.g. through the use of our forcings doi, but no citations so no information… there are not that many anymore?! Any ideas of how to get at this?


__________________________________________________

Prof. Michaela I. Hegglin (Nat. Sc. ETH)
Department of Meteorology
University of Reading
Brian Hoskins Building
Reading, RG6 6BB

Email  ***@***.***
Phone +44 118 378 6393
Web  https://research.reading.ac.uk/meteorology/people/michaela-hegglin/
__________________________________________________






On 20. Aug 2025, at 15:50, Vaishali Naik ***@***.***> wrote:



Out of curiosity, do we know how many models will use this ozone dataset to drive their CMIP7 simulations? Do we also know how many of these include internally generated QBO versus not?

**Comment by eleanororourke** [comment]
We have some responses from modelling centres from January 2025 who indicated whether they would/would not utilise the ozone and/or nitrogen forcing datasets. If useful I could send a note to them to ask whether they include internally generated QBO or not?

**Reply by vnaik60** [reply]
That would be very helpful. And also pointing them to this discussion would be great. Thank you @eleanororourke.

**Reply by JamesAnstey** [reply]
That would definitely be useful, thank you @eleanororourke.

For previous CMIP phases, [Richter et al. 2020](https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2019JD032362) assessed QBOs in historical runs from CMIP3,5,6. The number of models simulating a QBO in each phase was:

- CMIP3: none
- CMIP5: 5 
- CMIP6: 15

So for CMIP6, out of 30 models that had data available for this analysis (zonal-mean zonal wind & temperature) around the start of CMIP6, half of them simulated a QBO. I don't know what it will be for CMIP7, but perhaps around half of them (or maybe more than half), if this trend continues?

**Reply by durack1** [reply]
Ok this is great to know @JamesAnstey. And next logical question, how "realistic" were these CMIP5/6 QBOs, and how was "realism" measured - against obs, or ERA5 or something (maybe I should just read Richter et al., 2020...)

**Comment by hegglinm** [comment]
@JamesAnstey Thank you, this is indeed good to know! @eleanororourke I would be interested in knowing who indicated that they would use the ozone / nitrogen databases. Do you have this information handy? Thanks!

**Reply by eleanororourke** [reply]
The centres who indicated they need ozone and/or nitrogen were IITM, DWD, E3SM, IPSL, EC-Earth, MOHC, CCCma and AS-RCEC (email yesterday went to the science leads at these centres). Others who need it may not have responded to that survey though!

**Reply by GuangZeng08** [reply]
We (DWD) would use the ozone fields without QBO included for our DECK and ScenarioMIP simulations. Thanks a lot.

**Reply by juliearblaster** [reply]
The ACCESS model also needs ozone/nitrogen fields

**Reply by jmulcahy11** [reply]
The HadGEM3-GC5 model requires ozone forcing data. It has an internally generated QBO. For CMIP7 our preference would be for a dataset without a QBO signal applied to the ozone data as we feel this could have detrimental impacts in our model if the internal and externally applied QBO signals were out of phase. 

UKESM requires nitrogen deposition forcing data.

We need both of these forcing data as soon as possible (please!!) as we are planning to start our formal CMIP7 simulations soon.

Many thanks for all your efforts.

**Comment by timstockdale** [comment]
Just to add a comment about my experience using CMIP6 ozone as an input for SEAS6. The IFS uses a linearised ozone scheme HLO which takes account of the effect of temperature changes on net ozone production rates. Upper stratosphere ozone levels quickly reach equilibrium values, which are sensitive to  temperature.  The CMIP6 data show an increase in ozone up to 2100 in the upper stratosphere to values much above pre-industrial. This will be largely due to the CO2-induced cooling at these levels changing the equilibrium values of the ozone. However, for the HLO scheme this is a feedback we already account for and don't want to double count. I would have loved to have the stratospheric temperature values that corresponded to the CMIP6 ozone values, so that I could either build the changes in these temperatures into my model, or adjust the observed ozone values to be the equilibrium values (changing due to transport and chemistry) at present day temperatures. Since they were not provided as part of the dataset I had to make a handmade temperature correction field based on plots of CMIP6 stratospheric temperature change (averaged over the full set of CMIP6 models, not the pair used to create the ozone values), cross-checked with estimates of the structure of observed stratospheric temperature trends. (As a sanity check on my results, it was interesting to see how the CMIP6 ozone temperature changes could be decomposed into temperature-induced changes, changes related to changes in the Brewer-Dobson circulation, and the chemical influence of ODPs).  It would have been simpler and better if the (model-averaged) ozone values had been accompanied by the corresponding model-averaged temperatures from the same integrations.  Is this something that could be done with the CMIP7 data?   This would be helpful for anyone wanting to use a linearised ozone approach, which is far from perfect but does allow much improved relationships between ozone and the model simulation (specifically its own temperature and transport, including QBO-related variations) than imposing absolute fixed ozone values.

I don't want to add a lot of work for @hegglinm, but if the temperature data is at hand and easy to process in the same way as the ozone, it might be a nice complementary dataset for CMIP7, also in understanding drivers of the CMIP7-ozone changes.

**Reply by durack1** [reply]
> I don't want to add a lot of work for @hegglinm, but if the temperature data is at hand and easy to process in the same way as the ozone, it might be a nice complementary dataset for CMIP7, also in understanding drivers of the CMIP7-ozone changes.

Thanks for raising this @timstockdale. It would be great for folks to collate any additional feedback on this thread, and once we've had numerous eyeballs looking at these new data, plan to mint a v1.1 update to catch any issues, and hopefully (if it's possible) also generate some companion fields, like 3d temperature as you note could be very useful in mapping these data across to the consumption model levels etc

**Comment by durack1** [comment]
Hi all, just and FYI. The ozone data was published late last week, and can be downloaded/accessed from [here](https://esgf-node.ornl.gov/search?project=input4MIPs&activeFacets={%22mip_era%22:%22CMIP7%22,%22institution_id%22:%22FZJ%22}).

It would be great to get some eyes on this for testing, just to make sure that what has been provided checks out, and works for general use.

The nitrogen deposition is next, so hopefully have those finalized in a couple of weeks

**Reply by hwangacme** [reply]
@durack1 
Hi Paul, on the input4MIPs forcing data page, https://input4mips-cvs.readthedocs.io/en/latest/dataset-overviews/nitrogen-deposition/, the nitrogen deposition dataset appears to be ready, but it cannot be found on the ESGF server:
https://aims2.llnl.gov/search?project=input4MIPs&versionType=all&activeFacets=%7B%22source_id%22%3A%22FZJ-CMIP-nitrogen-1-0%22%7D
Any update or suggestion?

**Reply by znichollscr** [reply]
@hwangacme apologies that was an error in the docs after I got too eager testing things out

Fixed now, see https://github.com/PCMDI/input4MIPs_CVs/releases/tag/v6.7.4 and updated page: https://input4mips-cvs.readthedocs.io/en/latest/dataset-overviews/nitrogen-deposition/
